.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,9 +19,48 @@ If there is no corresponding issue number, fill in N/A where it says [fill_in_nu
 ## Proof that changes are correct
 
 <!--
-Add videos/screenshots of the user-facing interface in various display sizes (mainly phone, tablet, and desktop display size) to demonstrate that the changes made in this PR work correctly. Please also include videos/screenshots of the developer tools browser console, so that we can be sure that there are no console errors.
-If the PR changes the UI in any of the files listed in rtl_css.py (i.e, those that have a separate .rtl.css file for styling), make sure to add screenshots with the site language set to Arabic as well.
-If the changes in your PRs are autogenerated via a script and you cannot provide proof for the changes then please leave a comment "No proof of changes needed because {{Reason}}".
+Add videos/screenshots of the user-facing interface to demonstrate that the changes made 
+in this PR work correctly. if you work on some developer facing feature provide 
+the videos/screenshots of the developer-facing interface. 
+
+Please also include videos/screenshots of the developer tools 
+browser console, so that we can be sure that there are no console errors.
+
+If the changes in your PRs are autogenerated via a script and you cannot provide proof 
+for the changes then please leave a comment "No proof of changes needed because {{Reason}}" 
+and remove all the sections below.
+-->
+
+#### Proof of changes on desktop with slow/throttled network
+
+<!--
+Make sure to properly verify that everything works correctly, and that there are 
+no weird UI mistakes or other problems. Also, if there are any newly added fields, 
+try to fill them out and test that different inputs are correctly accepted/rejected.
+
+Throttle the network (to 3G) using the browser Developer Tools. There should be no
+performance or UI issues while the network is slow.
+
+Reference: 
+ - Chrome: https://css-tricks.com/throttling-the-network/
+ - Firefox: https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor/Throttling
+-->
+
+#### Proof of changes on mobile phone
+
+<!--
+In some cases this is not needed (e.g. for pages that we do not expect to 
+support mobile phone or when working on a backend feature).
+
+Feel free to use the Developer Tools emulator for this.
```

Comment:
Add a link that gives guidance on what the "Developer Tools emulator" is

Reply:
Done.